snuba/web/views.py

```diff
@@ -422,7 +422,9 @@ def health() -> Response:
 
     if status != 200:
         metrics.increment("healthcheck_failed", tags=metric_tags)
-
+    metrics.timing(
+        "healthcheck_timing", time.time() - start, tags={"thorough": str(thorough)}
```

Comment:
```suggestion
        "healthcheck.latency", time.time() - start, tags={"thorough": str(thorough)}
```

I think matches the naming convention of other `.timing` usages